packages/swagger/src/generators/OperationGenerator.ts

```diff
@@ -53,14 +54,22 @@ export abstract class OperationGenerator<TOptions extends { oas: Oas } = { oas:
   }
 
   getSchemas(operation: Operation): OperationSchemas {
+    const parameters = operation.getParameters()
+
+    const queryParams = parameters.filter((param) => param.in === 'query')
```

Comment:
param.in could be anything but in swagger documentation it feels like these are the main/only ones?
https://swagger.io/docs/specification/describing-parameters/